Sources/Rendering/OpenGL/RenderWindow/index.js

```diff
@@ -349,6 +349,12 @@ function vtkOpenGLRenderWindow(publicAPI, model) {
           model.vrSceneFrame = model.vrDisplay.requestAnimationFrame(
             publicAPI.vrRender
           );
+          // If Broswer is chrome we need to request animation again to canvas update
+          if (model.isChrome) {
```

Comment:
Let's use a global variable in that module (at the top of the file).

```
const IS_CHROME = navigator.userAgent.indexOf('Chrome') !== -1;
```

Reply:
Resolved, Thanks!